table/block_based/block_based_table_builder.h

```diff
@@ -92,10 +92,10 @@ class BlockBasedTableBuilder : public TableBuilder {
   // Get table properties
   TableProperties GetTableProperties() const override;
 
-  // Get checksum value
-  uint32_t GetFileChecksum() const override { return file_checksum_; }
+  // Get file checksum
+  std::string GetFileChecksum() const override { return file_checksum_; }
```

Comment:
Nit: for all table builder classes, the return type of `GetFileChecksum()` can actually be `const std::string&`, given that the file checksum won't change after file is written.